ios/FluentUI/Command Bar/MultilineCommandBar.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFMultilineCommandBarRow)
+public class MultilineCommandBarRow: NSObject {
+    var itemGroups: [CommandBarItemGroup] = []
+    var isScrollable: Bool
+
+    public init(itemGroups: [CommandBarItemGroup], isScrollable: Bool = false) {
+        self.itemGroups = itemGroups
+        self.isScrollable = isScrollable
+    }
+}
+
+@objc(MSFMultilineCommandBar)
+public class MultilineCommandBar: UIViewController {
+
+    // MARK: - Public methods
+
+    @objc public init(compactRows: [MultilineCommandBarRow], regularRows: [MultilineCommandBarRow]? = nil) {
```

Comment:
What happens if we pass in empty arrays in both? Would we need to add assertionFailures ?